style/aboutwikipedia.less

```diff
@@ -44,6 +44,11 @@
 			font-weight: bold;
 			font-size: 22px;
 			line-height: 26px;
+
+			@media ( min-width: 250px ) {
```

Comment:
what do you think we use it for

`@media @portrait` and `@media @landscape`

http://lesscss.org/#escaping

Reply:
Using a variable does sound like a good idea to try to limit the spread of magic numbers.

Reply:
Not a bad idea Huei, I introduced a `@landscape` variable now. Hoping that name won't be confusing (as these phones don't technically "support" landscape and adjust screen if a user turns it 90 degrees, right?).

The only other media query is a `min-width: 200px` used in `onboarding` and `aboutwikipedia`, does it make sense to set that as a `@portrait`variable?

Reply:
`@portrait` is useful! 

Would be more clear if we add the comment to `variable.style` to said the dimension for portrait 240x320 and landscape 320x240?